packages/react-server/package.json

```diff
@@ -24,50 +24,50 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "cls-q": "1.1.0",
-    "cookie": "0.1.2",
+    "cookie": "^0.3.1",
     "cookie-parser": "1.4.3",
-    "eventemitter3": "0.1.6",
-    "express-state": "1.2.0",
+    "eventemitter3": "^2.0.2",
+    "express-state": "^1.4.0",
     "lodash": "^4.16.4",
-    "mobile-detect": "1.3.0",
+    "mobile-detect": "^1.3.5",
     "q": "1.4.1",
     "querystring": "0.2.0",
-    "request-local-storage": "1.1.0",
-    "routr": "0.1.3",
-    "winston": "2.2.0"
+    "request-local-storage": "^1.1.1",
+    "routr": "^2.1.0",
```

Comment:
@gigabo The upgrade of `routr` from version 0.1.3 to >1.0 introduced some [breaking changes](https://github.com/yahoo/routr/blob/master/CHANGELOG.md#breaking-changes-1) that impact us.  These changes and the bug in #312 are related, so I'll track it there.